chanzuckerberg/idseq-web
# Susan/filteringby status

# Description

- filter entire project 
- Filter within a project
- filter within a search query.
- Search entire project
- Search within project
- Search filtered project

## Type of change
- [x ] New feature (non-breaking change which adds functionality)

# Test and Reproduce
Outline the steps to test or reproduce the PR here.

1. 

## Impacted Areas in Application
List general components of the application that this PR will affect:

Which pages?
- [x ] All Projects Page


Specific components 

* 

# Testing Script:

###Login Page
* Successfully login to the application and get taken to the project page

###All Project Page
* Switch between projects 
* Search for a sample
* Navigate through different pages of samples
* Select Upload new sample button
* Select sample to see generated report

###Sample Details Page
* Information is consistent from Project Page
* Notes are editable and save successfully 
* Can navigate to report page 

# Checklist:

- [ x] I have run through the testing script to make sure current functionality is unchanged
